cosmo_tester/framework/test_hosts.py

```diff
@@ -140,6 +140,54 @@ def upload_necessary_files(self):
     def ssh_key(self):
         return self._ssh_key
 
+    def get_remote_file(self, remote_path, local_path, use_sudo=True):
+        """ DUmp the contents of the remote file into the local path """
+
+        with self.ssh() as fabric_ssh:
+            fabric_ssh.get(
+                remote_path,
+                local_path,
+                use_sudo=use_sudo
+            )
+
+    def put_remote_file(self, remote_path, local_path, use_sudo=True):
+        """ Dump the contents of the local file into the remote path """
+
+        with self.ssh() as fabric_ssh:
+            fabric_ssh.put(
+                local_path,
+                remote_path,
+                use_sudo=use_sudo
+            )
+
+    def get_remote_file_content(self, remote_path, use_sudo=True):
+        tmp_local_path = os.path.join(self._tmpdir, str(uuid.uuid4()))
+
+        self.get_remote_file(remote_path, tmp_local_path, use_sudo)
+
+        with open(tmp_local_path, 'r') as f:
+            content = f.read()
+
+        os.unlink(tmp_local_path)
```

Comment:
These unlinks should probably be in a try|finally with an if os.path.exists guard?